hack/build-functest.sh

```diff
@@ -16,8 +16,9 @@ else
 	echo "GINKO binary found at $GINKGO"
 fi
 
+LDFLAGS="-X github.com/red-hat-storage/ocs-operator/version.Version=${CSV_VERSION}"
 
-"${GINKGO}" build "functests/${suite}/"
+"$GOBIN"/ginkgo build --ldflags "${LDFLAGS}" "functests/${suite}/"
```

Comment:
```suggestion
"${GINKGO}" build --ldflags "${LDFLAGS}" "functests/${suite}/"
```
This value is already set at Line no. 9. No need to change it.